### ISSUE TITLE
the "Loading..." row in tables is clickable and opens a modal

### ISSUE THREAD

**Issue by ramimanna** [aluo-x/813_site#32]

**Comment by ramimanna** [comment]
when this is clicked, a new document in the database can be made, but that document is empty and this throws and error and leaves it in the row is loading state forever :P

**Comment by nsylv** [comment]
oh wow, I wouldn't have thought to check that, nice catch!

probably we need to do something like disabling the table until it's loaded?

**Comment by ramimanna** [comment]
Yeah I think it's the handleClickRow happening because loading is in a tr
